bootstrap.php

```diff
@@ -97,7 +97,7 @@ public function htmlUnclosed()
     $styles = $this->getStyles();
     $app_id = WordPressEscaper::escAttr($settings['app_id']);
     $secret = WordPressEscaper::escAttr($settings['secret']);
-    $secure_mode = WordPressEscaper::escAttr($settings['secure_mode']);
```

Comment:
This settings option refers to a data storage in Wordpress. Simply replacing the name of the value is going to lead to retrieve nil values for current customers.